web/components/tasks/LicenseStatusReceipt.tsx

```diff
@@ -37,33 +37,63 @@ const activePermitTheme: PermitTheme = {
   headerIconColor: "text-success",
 };
 
+const grayPermitTheme: PermitTheme = {
```

Comment:
This is a nitpick and I'm sorry but can we be consistent about 'grey' vs 'gray'? Personally I prefer 'grey' but it looks like njwds uses 'gray' so 🤷